lib/browser/browser.js

```diff
@@ -17,11 +18,12 @@ module.exports = class Browser {
     }
 
     captureViewportImage(page) {
-        return this._camera.captureViewportImage(page);
```

Comment:
`browser.captureViewportImage` will be called
- on calibration
- on any error
- few times during capturing if `compositeImage` is `true`

So this is not the good place to insert the delay.
The best place probably would be the [CaptureSession.capture](https://github.com/gemini-testing/gemini/blob/v5.1.2/lib/capture-session/index.js#L41-L49) (which will be preformed in worker) or even [CaptureSession.prepareScreenshot](https://github.com/gemini-testing/gemini/blob/v5.1.2/lib/capture-session/index.js#L36-L39) (which will be performed in master process)

Reply:
Actually the goal is that the delay is executed multiple times when `compositeImage` is `true`. That is why I added it in this position.

E.g. we have a scroll to top button which is faded in when the user scrolls down. As the browser is scrolled down when `compositeImage` is `true`, the delay needs to be executed after scrolling to ensure that the animation has finished. As far as I can see adding them to `CaptureSession.capture` or `CaptureSession.prepareScreenshot` will only delay once before creating the composite screenshot and thus create screenshots with an inconsistently faded in scroll to top button.

Reply:
Ok.